django/example/settings.py

```diff
@@ -51,8 +51,13 @@
         "BACKEND": "django.template.backends.django.DjangoTemplates",
         "DIRS": [],
         "APP_DIRS": True,
-        "OPTIONS": {
-            "context_processors": ["django.contrib.auth.context_processors.auth"]
+        'OPTIONS': {
+            'context_processors': [
+                'django.template.context_processors.debug',
+                'django.template.context_processors.request',
+                'django.contrib.auth.context_processors.auth',
+                'django.contrib.messages.context_processors.messages',
```

Comment:
I can't understand the exact problem from the stack overflow question.

What was broken and how it was fixed with these lines?